bin/sl-run.txt

```diff
@@ -22,6 +22,9 @@ Options:
                      Disable timestamping of worker log lines by supervisor.
   --no-timestamp-supervisor
                      Disable timestamping of supervisor log messages.
+  --no-log-decoration
```

Comment:
I'm ok with it as is. Could maybe extend the description to say something like ".. non-timestamp decorations, including cluster id/pid" so that it covers any future tags that may be added.